src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/nbthandlers/GT_MetaTileEntity_Hatch_NbtConsumable.java

```diff
@@ -120,7 +120,8 @@ public final void updateSlots() {
 			if (i <= getSlotID_LastInput()) {
 				fillStacksIntoFirstSlots();
 			}
-		}            
+		}
+		tryFillUsageSlots();
```

Comment:
Not happy about this, but it's `private` and it calls updateSlots() I think ... which is this. But it works, it correctly loads the next catalyst, and it does NOT load two or something like that.

Reply:
Just turn tryFillUsageSlots into public. It won't hurt.

Reply:
ok I can do that

Reply:
Reverting because that didn't work .... the bug came back

Reply:
wait maybe I have the dumb